site/src/Consumer/AbstractNotificationConsumer.php

```diff
@@ -72,6 +81,11 @@ public function setNotification(Notification $notification)
         $this->notification = $notification;
     }
 
+    public function setRecipient(User $user)
+    {
+        $this->recipient = $user;
+    }
+
```

Comment:
You have this information already available in `$notification`, no need for a separate `$recipient` variable. (Actually, it's harmful to have a second copy of the same information, as the two could get out of sync.)

Reply:
Yep